CODEOWNERS

```diff
@@ -154,6 +154,7 @@
 /drivers/*/*lpc11u6x*                     @mbittan @simonguinot
 /drivers/adc/                             @anangl
 /drivers/adc/adc_stm32.c                  @cybertale
+/drivers/adc/*npcx*                       @MulinChao
```

Comment:
You might want to consider replacing all the driver-specific lines with
```
/drivers/*/*npcx*                       @MulinChao
```

Reply:
If someone in our team is in charge of the other driver, for example, ps2 driver, CODEOWNERS file might look like:
/drivers/*/*npcx*                       @MulinChao
/drivers/ps2/*npcx*                   @Someone else
Is it okay?

Reply:
In that case, then stick with the driver-specific lines. It initially looked like you were taking all of the ncpx drivers.

Reply:
Currently, I'm the only one for porting Nuvoton NPCX drivers. But more resources in Nuvoton will join this project soon.

Reply:
Done. I added it in the last CL.